stdlib/gc.mli

```diff
@@ -149,7 +149,42 @@ type control =
         out variations in its workload. This is an integer between
         1 and 50.
         Default: 1. @since 4.03.0 *)
-}
+
+    custom_major_ratio : int;
+
```

Comment:
This empty line explains the CI failure:
````
File "/home/travis/build/ocaml/ocaml/stdlib/gc.mli", line 155, characters 4-580:
Error (warning 50): unattached documentation comment (ignored)
````

Reply:
Yes yes, I'm fixing it.